### ISSUE TITLE
Pretty printing duplicates lines

### ISSUE THREAD

**Issue by x-yuri** [mina-deploy/mina#276]
`Gemfile`:

```ruby
source 'https://rubygems.org'
gem 'mina', github: 'mina-deploy/mina', ref: '9b97f8ebc6ddf1e80b295921edfb46907b925726'   # current master
```

`Minafile`:

```ruby
# change 102 in the script to result of `tput cols`
task :t1 do
  (0..102).each { |i|
    print_char (i % 10).to_s
  }
  print_char "\n"
  print_str (0..102).map { |i| (i % 10).to_s }.join + "\n"
end
```

`$ bundle exec mina t1` (with `tput cols` == 102):

```
       01234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234
       01234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234
567890
       01234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234
567890

```

**Comment by x-yuri** [comment]
Apparently, this is owing to [this commit](https://github.com/mina-deploy/mina/commit/88b9d8f). Which fixes displaying password prompts, when you're expected to enter a password on the same line. But when terminal width is less then line to be printed, [`print_clear`](https://github.com/mina-deploy/mina/blob/v0.3.2/lib/mina/output_helpers.rb#L50-52) thing doesn't work.

**Comment by pramodshinde** [comment]
@x-yuri I checked with current master for given rake task its getting printed correctly

`$bundle exec mina t1`
```ruby
       0123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012
       0123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012
```
I think this issue can be closed.

**Comment by x-yuri** [comment]
No, I still can reproduce it with current master (`fc02c9385c7f7760aa39f8d46edbda8bcd3512a9`). It must be not really clear, that you've got to change `102` in the test script with your actual screen width (`tput cols`). Anyways, it must be a number big enough for the numbers to not fit the line.

**Comment by pramodshinde** [comment]
@x-yuri My terminal shows `130` on `tput cols`, Can you tell me how to change it to `102` also when you say '...in test script which test script with your actual screen width' are you referring here? Am I missing something?

**Comment by x-yuri** [comment]
`tput cols` reports number of columns your terminal has. And you're not supposed to change it. You're supposed to replace 102 in the test script with what `tput cols` outputs. So, for 130 columns test script looks like:

```
task :t1 do
  (0..130).each { |i|
    print_char (i % 10).to_s
  }
  print_char "\n"
  print_str (0..130).map { |i| (i % 10).to_s }.join + "\n"
end
```

**Comment by d4be4st** [comment]
This should be fixed i new 1.0.0 release
